eng/pipelines/templates/steps/apiview-review-gen.yml

```diff
@@ -16,19 +16,24 @@ parameters:
     default: ''
 
 steps:
-- task: Powershell@2
+- task: AzurePowerShell@5
   displayName: 'Generate APIView Token files'
+  continueOnError: false
```

Comment:
This shouldn't be needed. Setting this to false is the default

Reply:
Removed in second commit.